### PR TITLE
Add sebastian to members.

### PR DESCRIPTION
**Pull request by sprillo**
Test plan: serve locally.

### CODE REVIEW

File: _data/members.yml [hunk 1]
```diff
@@ -136,11 +136,11 @@
...
+  image: sebastian_prillo.jpg
   altimage:
   position: Graduate student (EECS); joint with Yun Song
-  email:
-  website:
-  scholar:
-  twitter:
+  email: sprillo (at) berkeley.edu
+  website: http://people.eecs.berkeley.edu/~sprillo/
+  scholar: dvJiM_kAAAAJ
```
Comment (adamgayoso) [inline _data/members.yml]: I think this needs the last 3 characters like mine and Nir's and Romain's

File: _data/members.yml [hunk 2]
```diff
@@ -136,11 +136,11 @@
 
 - name:  Sebastian Prillo
   degrees:
-  image: john_d.jpg
+  image: sebastian_prillo.jpg
   altimage:
   position: Graduate student (EECS); joint with Yun Song
```
Comment (adamgayoso) [inline _data/members.yml]: should it be Yun S. Song?

File: _data/members.yml (hunk 1 above)
Comment (sprillo) [reply _data/members.yml]: @adamgayoso really? `&hl` sounds like the language parameter, as in `https://scholar.google.com/citations?hl=en&user=dvJiM_kAAAAJ`. If I use `&hl` as you suggest I get redirected to `https://scholar.google.com/citations?user=dvJiM_kAAAAJ&hl=`. Does this make my google scholar landing page default to english? I'll push the change though, but was just curious to hear why it's needed!

Comment (adamgayoso) [reply _data/members.yml]: oh ok I have no clue honestly haha FOLLOWING PATTERNS

File: _data/members.yml (hunk 2 above)
Comment (sprillo) [reply _data/members.yml]: Ah, good point, I think what you suggest is better. Thanks! I'll update this.

**Review by adamgayoso — APPROVED** [review]

**Pull request merged**